### PR TITLE
payment_info examples & order status renaming

### PR DESCRIPTION
**Pull request by aljazceru**

### CODE REVIEW

**Comment by TaprootFreak** [comment]
I'm not saying I'm against it, and I don't want to stand in anyone else's way.

But, we at DFX don't want to offer that. At least not at the beginning. Maybe later when we know the use case well, but not now. There are massive risks to building in a service natively, and we can't judge today if that's a good idea. 

I think we should start with a payment URL solution now and then check a native implementation later in a second step. 

If someone else wants to do it directly I am not against it. DFX will not do it at the moment.

**Comment by aljazceru** [comment]
> I'm not saying I'm against it, and I don't want to stand in anyone else's way.
> 
> But, we at DFX don't want to offer that. At least not at the beginning. Maybe later when we know the use case well, but not now. There are massive risks to building in a service natively, and we can't judge today if that's a good idea.
> 
> I think we should start with a payment URL solution now and then check a native implementation later in a second step.
> 
> If someone else wants to do it directly I am not against it. DFX will not do it at the moment.

@TaprootFreak  there is a payment_url example in the code for the credit card flow, we could make it more generic if the payment link is the only option for all payment methods but the structure is the same

**Comment by TaprootFreak** [comment]
> @TaprootFreak there is a payment_url example in the code for the credit card flow, we could make it more generic if the payment link is the only option for all payment methods but the structure is the same

yes we can do that.

File: FLS01/README.md [hunk 1]
```diff
@@ -186,16 +187,39 @@ Response:
...
+```
 
-### orders 
+### order-status 
 Get order status 
 
 Request:
 ```
-POST /order
+POST /order-status
```
Comment (TaprootFreak) [inline FLS01/README.md:222]: POST not GET?

**Comment by TaprootFreak** [comment]
I think "payment_url" should be a fixed part of the specification. It should not maybe be returned or not. It should be fixed to the protocol and always returned. If the broker doesn't support it then he can return it empty. 

All big providers work with it, so it should be fixed part of the protocol, same as the other things.

**Comment by aljazceru** [comment]
@TaprootFreak added better support for payment_url with explanations

**Review by TaprootFreak — APPROVED** [review]

**Comment by TaprootFreak** [comment]
Agreed.

**Pull request merged**